app/models/broker_stock.rb

```diff
@@ -1,2 +1,8 @@
 class BrokerStock < ApplicationRecord
+  validates :user_id, presence: true
+  validates :stock_id, presence: true
+  validates :company, presence: true
+  validates :ticker, presence: true
+  validates :price, presence: true
+  validates :created_at, presence: true
```

Comment:
Rails always gives `created_at` a value. You don't have to validate this attribute